daemon/CryptoImpl/cryptorequestprocessor.cpp

```diff
@@ -446,7 +454,12 @@ Daemon::ApiImpl::RequestProcessor::generateStoredKey_afterPreCheck(
             ikdRequest.setPluginName(keyTemplate.identifier().storagePluginName());
             ikdRequest.setOperation(Sailfish::Secrets::InteractionParameters::DeriveKey);
             ikdRequest.setAuthenticationPluginName(uiParams.authenticationPluginName());
-            ikdRequest.setPromptText(uiParams.promptText());
+            //: This will be displayed to the user, prompting them to enter a passphrase from which a key will be derived.  %1 is the key name, %2 is the collection name, %3 is the plugin name.
+            //% "An application wants to store a new key named %1 within collection %2 in plugin %3.  Enter a passphrase from which the key will be derived."
```

Comment:
"..in plugin %3.  Enter a pass..." -> ..in plugin %3. Enter a pass..." ? We don't do double spaces after comma in translation strings?

Reply:
Oops, I will fix all those cases.  Old habits for manual typesetting ;-)